exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+// var numberOfStudents = 15;
+// var numberOfMentors = 8;
+var preciseAge = 30.612437;
+var result = preciseAge / 10;
+var resultWhole = Math.round(result);
+console.log(resultWhole + "%");
```

Comment:
Please have look again at the Readme file because the instruction expects something else.
Please let me know if you find any straggling!
 
## Exercise
- Using the variables provided in the exercise calculate the percentage of mentors and students in the group
## Expected result

```
Percentage students: 65%
Percentage mentors: 35%